data/crac/crac-api/src/main/java/com/farao_community/farao/data/crac_api/Cnec.java

```diff
@@ -35,9 +35,9 @@
      * positive, it means that the limits of the Threshold are respected. If it is negative,
      * it means that that a limit of the Threshold has been overcome.
      *
-     * margin = min(maxThreshold - actualValue, actualValue - minThreshold)
+     * margin = min(maxThreshold - actualFlow, actualFlow - minThreshold)
      */
-    double computeMargin(Network network, Unit requestedUnit);
+    double computeMargin(double actualFlow, Unit flowUnit);
```

Comment:
It was value because CNEC is supposed to be working with voltage or angle monitoring...

Reply:
So actualValue and valueUnit?